docs/profiling.rst

```diff
@@ -1,6 +1,10 @@
 Data Profiling
 ==============
 
+.. note::
+   These features are no longer supported in the new FAB-based webserver
```

Comment:
Nit:  singular form would be better( `This feature is no longer`).  And add a note to indicate this section should be removed once we migrate to rbac UI.